config/checkstyle_checks.xml

```diff
@@ -424,6 +424,14 @@
       <message key="matchxpath.match"
                value="Avoid using deprecated method 'DetailAst.branchContains()'."/>
     </module>
+    <module name="MatchXpath">
+      <property name="id" value="noUsageOfGetFileContents()Method"/>
```

Comment:
please make name `noUsageOfGetFileContentsMethod` to be be selectable by double click on it by mouse and ....

Reply:
> please make name `noUsageOfGetFileContentsMethod` to be be selectable by double click on it by mouse and ....

and what @romani ?

Reply:
Please remove `()` from name